watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/suboxic_activated_sludge_process/suboxic_ASM_ui.py

```diff
@@ -0,0 +1,571 @@
+    )
+
+    # System metrics
+    exports.add(
+        obj=fs.costing.LCOT,
+        name="Levelized cost of treatment",
+        ui_units=fs.costing.base_currency / pyunits.m**3,
+        display_units="$/m3 of feed",
+        rounding=2,
+        description="Levelized cost of treatment including operating and capital costs",
+        is_input=False,
+        is_output=True,
+        output_category="Levelized cost metrics",
+    )
+    exports.add(
+        obj=fs.costing.LCOW,
+        name="Levelized cost of water",
+        ui_units=fs.costing.base_currency / pyunits.m**3,
+        display_units="$/m3 of product",
+        rounding=2,
+        description="Levelized cost of water including operating and capital costs",
+        is_input=False,
+        is_output=True,
+        output_category="Levelized cost metrics",
+    )
+
+    # Normalized metrics
+    total_capital_norm = fs.costing.total_capital_cost / fs.feed.properties[0].flow_vol
+    exports.add(
+        obj=total_capital_norm,
+        name="Total capital",
+        ui_units=fs.costing.base_currency / (pyunits.m**3 / pyunits.day),
+        display_units="$/(m3/day)",
+        rounding=1,
+        description="Normalized total capital costs accounting for indirect "
+        "capital and installation - [total capital costs/feed flow rate]",
+        is_input=False,
+        is_output=True,
+        output_category="Normalized cost metrics",
+    )
+    direct_capital_norm = (
+        fs.suboxicASM.costing.capital_cost
+        / fs.costing.TIC
+        / fs.feed.properties[0].flow_vol
+    )
+    exports.add(
+        obj=direct_capital_norm,
+        name="Direct capital",
+        ui_units=fs.costing.base_currency / (pyunits.m**3 / pyunits.day),
+        display_units="$/(m3/day)",
+        rounding=1,
+        description="Normalized direct capital costs - [total direct capital "
+        "costs/feed flow rate] ",
+        is_input=False,
+        is_output=True,
+        output_category="Normalized cost metrics",
+    )
+    elec_operating_norm = (
+        fs.costing.aggregate_flow_costs["electricity"] / fs.costing.annual_water_inlet
+    )
+    exports.add(
+        obj=elec_operating_norm,
+        name="Electricity",
+        ui_units=fs.costing.base_currency / pyunits.m**3,
+        display_units="$/m3 of feed",
+        rounding=4,
+        description="Normalized electricity cost - [annual electricity costs/annual "
+        "feed flow rate]",
+        is_input=False,
+        is_output=True,
+        output_category="Normalized cost metrics",
+    )
+
+    # performance metrics
+    recovery_vol = (
+        fs.product_H2O.properties[0].flow_vol / fs.feed.properties[0].flow_vol
+    )
+    exports.add(
+        obj=recovery_vol,
+        name="Volumetric recovery",
+        ui_units=pyunits.dimensionless,
+        display_units="m3 of product/m3 of feed",
+        rounding=4,
+        description="Normalized heating cost - [annual heating costs/annual feed "
```

Comment:
```suggestion
        description="Normalized volumetric recovery"
```
I think this typo slipped through the cracks in the metab_ui example.